tsdb/exemplar.go

```diff
@@ -150,10 +167,13 @@ func (ce *CircularExemplarStorage) indexGc(cbe *circularBufferEntry) {
 	ce.index[l] = &indexEntry{i, ce.index[l].last}
 }
 
-func (ce *CircularExemplarStorage) AddExemplar(l labels.Labels, e exemplar.Exemplar) error {
+func (ce *CircularExemplarStorage) AddExemplar(l labels.Labels, e exemplar.Exemplar) (err error) {
```

Comment:
was the named return value change required for something?

Reply:
Not anymore, sorry - I still need to find out if we want to instrument for this extra stuff or not